test/functional/page_objects/vega_chart_page.ts

```diff
@@ -17,7 +17,6 @@
  * under the License.
  */
 
-import { Key } from 'selenium-webdriver';
```

Comment:
Using `browser` service instead so that we can easily migrate to other test library